### DISCUSSION TITLE
Extended R language ligature support

### DISCUSSION THREAD

**Discussion by Rieleniel** [tonsky/FiraCode#1618] [Ideas]
Dear creator(s),

I started using FiraCode over a year ago, and so far, I enjoy it very much for python programming! Today, I was curious to see if it would enhance my R coding experience, and there are a few remarks I would like to add:

- In R, certain defusing operators can be useful for function coding. You have covered the standard injection operator and the base pipe operator (`!!` and `|>`, respectively), but you have not yet covered other operators (`%>%`, `%$%`, `%T>%` and `%<>%`) and more advanced cases of injection (`!!!`). It would also be delightful to see a ligature for the embrace operator (`{{` and `}}`) as using an embrace ligature could improve visibility.

**Comment by tonsky** [comment]
Thanks! There’s an issue about %>% already, main takeaway is that it’s not clear how to draw it so that it stays recognizable yet improves on default rendering.

`!!!` and `{{` are probably no-go as they will conflict for many other languages, including plain text
